tests/test_cli/test_run.py

```diff
@@ -754,7 +754,7 @@ def setup_class(cls):
 
         shutil.copytree(
             Path(CUR_PATH, "data", "exception_skill"),
-            Path(cls.t, cls.agent_name, "skills", "exception"),
+            Path(cls.t, cls.agent_name, "vendor", "fetchai", "skills", "exception"),
```

Comment:
In this test, the exception skill was not placed correctly. Since the author is `fetchai` it should have been put in the vendor directory.